README.md

```diff
@@ -1,42 +1,72 @@
+    >
+    <img
+      width="512px"
+      src="assets/wordmark/wordmark+slogan-light.svg"
+    >
+  </picture>
+  <br>
+  <a href="https://revenge-mod.github.io/">
+       <picture>
+           <source height="32px" media="(prefers-color-scheme: dark)" srcset="assets/logo/logo-dark.svg" />
+           <img height="32px" src="assets/logo/logo-light.svg" />
+       </picture>
+   </a>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
+   <a href="https://discord.com/invite/ddcQf3s2Uq">
+       <picture>
+           <source height="32px" media="(prefers-color-scheme: dark)" srcset="https://user-images.githubusercontent.com/13122796/178032563-d4e084b7-244e-4358-af50-26bde6dd4996.png" />
+           <img height="32px" src="https://user-images.githubusercontent.com/13122796/178032563-d4e084b7-244e-4358-af50-26bde6dd4996.png" />
+       </picture>
+   </a>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
+   <a href="https://github.com/revenge-mod">
+       <picture>
+           <source height="32px" media="(prefers-color-scheme: dark)" srcset="https://i.ibb.co/dMMmCrW/Git-Hub-Mark.png" />
+           <img height="32px" src="https://i.ibb.co/9wV3HGF/Git-Hub-Mark-Light.png" />
+       </picture>
+   </a>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
 
-## Installing
+   </a>
+</p>
+
+A client modification for Discord mobile, a continuation of [Bunny](https://github.com/pyoncord/Bunny).
+
+## 💪 Features
+
+- **🔌 Plugins**: Extend Discord with custom features
+- **🎨 Themes & Fonts**: Customize Discord's appearance to your liking
+- **🧪 Experiments**: Try out Discord's new features before they're rolled out
+
+## ⬇️ Download
 
 ### Android
 
-- **Root** with Xposed - [RevengeXposed](https://github.com/revenge-mod/RevengeXposed/releases/latest)
-- **Non-root** - [Revenge Manager](https://github.com/revenge-mod/RevengeManager/releases/latest)
+- **🩹 Root with Xposed**: [RevengeXposed](https://github.com/revenge-mod/revenge-xposed/releases/latest)
+- **📵 Non-root**: [Revenge Manager](https://github.com/revenge-mod/revenge-manager/releases/latest)
 
 ### iOS
-- [**RevengeTweak**](https://github.com/revenge-mod/RevengeTweak) - Get prebuilt rootful and rootless `.deb` files or the prepatched `.ipa `
+- [**RevengeTweak**](https://github.com/revenge-mod/revenge-tweak): Prebuilt rootful and rootless `.deb` files or the prepatched `.ipa `
+
 
-## Building
-1. Install a Revenge loader with config support (any mentioned in the [Installing](#installing) section).
-2. Go to **Settings** > **General** and enable **Developer Settings**.
-3. Clone the repository
+## 📚 Everything else
 
-    ```sh
-    git clone https://github.com/revenge-mod/Revenge.git
-    ```
+### 📙 Contributing
 
-4. Install dependencies
+Thank you for considering contributing to Revenge.
+You can find the contribution guidelines [here](CONTRIBUTING.md).
 
-    ```
-    pnpm i
-    ```
+### 🛠️ Building
 
-5. Build Revenge's code
+To build a Revenge, you can follow the [documentation](/docs).
```

Comment:
```suggestion
To build Revenge, you can follow the [documentation](/docs).
```

Reply:
I think it's best to rewrite the whole file, not in this PR though.

Reply:
The issue can be addressed here for the time being